docs/metrics/v3.md

```diff
@@ -1,397 +1,409 @@
 # Metrics Version 3
 
-In metrics version 3, all metrics are available under the endpoint:
+In metrics version 3, all metrics are available under the following base endpoint:
 
 ```
 /minio/metrics/v3
 ```
 
-however, a specific path under this is required.
+To query metrics of a specific type, append the appropriate path to the base endpoint.
+You **must** append a path, as querying the base endpoint alone does not return metrics.
 
 Metrics are organized into groups at paths **relative** to the top-level endpoint above.
 
+Metrics are also available using the [MinIO Admin Client](https://min.io/docs/minio/linux/reference/minio-mc-admin.html) and the `mc admin prometheus metrics` command. For more information, see [Metrics and Alerts](https://min.io/docs/minio/linux/operations/monitoring/metrics-and-alerts.html) in the MinIO Documentation.
+
 ## Metrics Request Handling
 
-Each endpoint below can be queried at different intervals as needed via a scrape configuration in Prometheus or a compatible metrics collection tool.
+Each endpoint can be queried as needed via a scrape configuration in Prometheus or a compatible metrics collection tool. You should schedule scrape operations so a prior scrape completes before the next one begins.
 
-For ease of configuration, each (non-empty) parent of the path serves all metric endpoints that are at descendant paths. For example, to query all system metrics one needs to only scrape `/minio/metrics/v3/system/`.
+For ease of configuration, each (non-empty) parent of the path serves all the metric endpoints at its child paths. For example, to query all system metrics scrape `/minio/metrics/v3/system/`.
```

Comment:
This might make more sense if we provide a sample tree. Spitballing.

```
Each metric endpoint may support multiple child endpoints. For example, the `/v3/system/` metric has five child groups as `/v3/system/<child>`. Querying the parent endpoint returns metrics for all child groups.
Querying a child group returns only metrics for that child.
```